product_docs/docs/pem/10/installing/ha/index.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: "Installing Postgres Enterprise Manager in HA Patterns"
+navTitle: "High Availability PEM Installation"
+
+redirects:
+
+navigation:
+  - setup_ha_using_efm
+  - efm_s1
+  - patroni
+  - user_settings
+---
+
+Before deploying PEM in an HA pattern, please familiarize yourself with the high-level guidance in  [High Availability Patterns for PEM Deployment](/pem/latest/considerations/ha_pem.mdx).
+
+The following pages provide detailed instructions for some example HA PEM architectures. They are not intended to be an exhaustive guide to all possible patterns.
+
+- [Deploying HA PEM with colocated web application and backend using EDB Failover Manager and Virtual IPs](setup_ha_using_efm.mdx)
+- [Deploying HA PEM with separate web application and backend using EDB Failover Manager and Virtual IPs](efm_s1.mdx)
+- [Deploying HA PEM with Patroni](patroni.mdx)
```

Comment:
Are we going to add this file, "patroni.mdx"?

Reply:
Yes, but perhaps better to do so in another PR so we can merge this one sooner